brands/tourism/hotel.json

```diff
@@ -425,6 +486,15 @@
       "tourism": "hotel"
     }
   },
+  "tourism/hotel|JW Marriott": {
```

Comment:
Add a matching name or `official_name` for `JW Marriott Hotels`.

Reply:
Fixed